src/main/java/io/codelex/classesandobjects/practice/Exercise10/BankAccount.java

```diff
@@ -0,0 +1,36 @@
+package io.codelex.classesandobjects.practice.Exercise10;
+
+// Each BankAccount object represents one user's account
+// information including name and balance of money.
+
+public class BankAccount {
+    public String name;
+    public double balance;
+
+
+    public BankAccount(String name, double balance) {
+        this.name = name;
+        this.balance = balance;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public double getBalance() {
+        return balance;
+    }
+
+
+    public final void deposit(double amount) {
+        balance += amount;
+    }
+
+    public final void withdraw(double amount) {
+        balance -= amount;
+    }
+
+    public String toString(String name, double balance) {
+        return name + ", " + String.valueOf(balance);
```

Comment:
` If the balance is negative, put the - sign before the dollar sign. Also, always display the cents as a two-digit number. For example, if the same object had a balance of -17.5`